less/components/jtable.less

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------------
+//
+//  JTable and related components
+//
+//--------------------------------------------------------------------------
+
+//--------------------------------------
+//  Vars / Mixins
+//--------------------------------------
+
+@table-left-right-padding: 24px;
+@table-column-spacing: 32px;
+// !!!IMPORTANT!!! Don't change the above without changing the constants in JTable.jsx to match!
+
+// FIXME: We need to sort out the mess of different colours we've got all over the place
+@table-border-color: #e6eaee;
+@table-header-color: rgba(53, 64, 82, 0.5);
+@table-hover-bg: #fafbfd;
```

Comment:
@michaelneale @cliffmeyers I'm trying to introduce some blue tint greys to reduce the harshness of the straight greys, with the intent to replace the current greys.

We/I need to find time to dedicate to solidifying style guide... whilst trying to iterate and improve upon what we currently have.